src/assets/fonts.tsx

```diff
@@ -0,0 +1,11 @@
+import { injectGlobal } from 'styled-components'
+
+injectGlobal`
+    @font-face {
+        font-family: 'artsy-icons';
```

Comment:
@craigspaeth has good arguments against an icon font, the only thing I have to add is that iOS does not natively support SVG, but I’m sure there’s a way we can make that work.